files/adfc-hh-connect.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Aufbau einer VPN-Verbindung zum ADFC-HH UCS-MASTER
+
+
+# Uptime & Hostname
+UT=$(uptime | cut -d ' ' -f 4)
+HOST=$(hostname)
+
+
+# Alte Fehlermeldungen löschen
+find /$HOST* -mtime 1 -exec rm {} \;
+
+
+# Herstellen der VPN-Verbindung
+vpnc adfc-hh.conf
+sleep 5
+
+
+# Test und Fehlerbehandlung, falls VPN-Tunnel nicht aufgebaut wurde
+ping -c 3 192.168.123.32 #2>&1> /dev/null
```

Comment:
Für 192.168.123.32 eine Variable definieren?

Reply:
Gute Idee! Ist umgesetzt.

Reply:
Aber nicht in diesem Skript